Formula/terraform.rb

```diff
@@ -35,6 +39,18 @@ def install
     system "go", "build", *std_go_args(ldflags: "-s -w")
   end
 
+  def caveats
+    <<~EOS
+      Terraform has been deprecated in homebrew-core. We will not accept any new releases.
+      The next release will chage to a non-open-source license:
+      https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license
+      See our documentation for acceptable licences:
+      https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae
+
+      This also means that terraform might get disabled and phased out at one point.
```

Comment:
```suggestion
```

Reply:
Since this will follow the normal deprecation/disable prodecure, I don't think we need this.